README.md

```diff
@@ -18,6 +18,7 @@
 
 - **Variable formatting** ([instructions](https://nestjs-i18n.com/guides/formatting))
 - **Nested** translations ([instructions](https://nestjs-i18n.com/guides/nested))
+- **Deep** folders ([instructions](https://nestjs-i18n.com/guides/deep))
```

Comment:
I don't think it needs a whole separate page for it. Because it's basically an option for the `JSONLoader`. :)

Reply:
That's fine too. Where should I document it instead?

Reply:
I think it could be a section within the `formatting` page?

Reply:
Ow oops sorry, I mean within the `loaders` page.

Reply:
Done: https://github.com/toonvanstrijp/nestjs-i18n/pull/419/commits/537584e240ebf00dff815988979e76bcf0caf8cc